### ISSUE TITLE
[FEATURE] Clearly show in the documentation that firewalld will be installed (or don't install firewalld)

### ISSUE THREAD

**Issue by ShaunMaher** [saltstack-formulas/mongodb-formula#91]
### Is your feature request related to a problem?

I think the readme should clearly shout out that the "firewalld" package will be installed and that, by doing so, a default set of firewall rules will be applied.  I'll spare you a story but it took me quite a while to work out why I had lost access to several services that were completely unrelated to MongoDB by using this formula.

I might even humbly suggest that firewall package installation might be out-of-scope for a formula for MongoDB.  If it configures a firewall that is already installed, that seems appropriate.  Alternatively make installing firewalld optional and disabled by default.

### Describe the solution you'd like

Either edit README.rst to include a clear warning that firewalld is installed, and the impact that may have or, make the installation of firewalld optional and disabled by default.

**Comment by daks** [comment]
Hello,

your issue makes sense, this formula must not by default installs a specific firewall management tool.

I checked on our servers (we use this formula for very basic mongodb deployment) and firewalld is not installed. I searched on at the code and it seems that this line https://github.com/saltstack-formulas/mongodb-formula/blob/master/mongodb/defaults.yaml#L12 makes firewalld not be installed by default.
